pytroll_collectors/harvest_schedules.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 - 2021 Pytroll developers
+#
+# Author(s):
+#
+#   Trygve Aspenes <trygveas@met.no>
+#   Panu Lahtinen <panu.lahtinen@fmi.fi>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Harvest schedules. Download schedules from EUM and parse to limit gatherer"""
```

Comment:
If I understand correctly, this module parses EUM schedules. Could the name of the module and/or functions reflect this?

Reply:
Hm, good point.

Reply:
I think its enough to make the module reflect this. Then it will be easy to add new modules downloading and parsing other schedules using the same function name. And also for the user to provide custom modules.